tests/factories/user.py

```diff
@@ -97,3 +98,45 @@ async def users(
     async with async_session_factory() as async_session:
         await async_session.execute(query)
         await async_session.commit()
+
+
+@pytest_asyncio.fixture
```

Comment:
```suggestion
@pytest_asyncio.fixture(params=[NamespaceUserRole.DEVELOPER])
```
Так будет и задано какое-то конкретное значение для роли (как я понял, если случайно использовать фикстуру без роли, вылетит исключение), и плюс будет понятнее, чему соответствует значение в коде самой фикстуры